app/example-page/page.tsx

```diff
@@ -0,0 +1,11 @@
+import ExampleComponent from '../../components/ExampleComponent/ExampleComponent';
```

Comment:
When it comes to imports i want to suggest that we separate packages/external modules from modules within our codebase with empty line like that

`import Link from Next.js`
// here is an empty line that separates these imports
`import OurComponent from '@/components/...'`
`import schema from '@/lib/schemas'`

idk i feel like this is more readable, what do you think

Reply:
Yup, its a good idea